docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
         target: /opt/repos
         read_only: true
         consistency: delegated
+      - indexes:/var/lib/gitbase/index
```

Comment:
1. space
2. let's call it `gitbase_indexes` or something like that. `indexes` is way too general name.

Reply:
> space

what do you mean?

Reply:
ah. My bad. It is just below lines with syntax highlighted but this line not. But actually it's correct.

Reply:
:+1: name changed